content/en/docs/Getting started/_index.md

```diff
@@ -13,82 +13,132 @@ description: >
+## Option 1: Clone the Docsy example site
 
-Copy the [example site's repo](https://github.com/google/docsy-example). Using this approach also gives you a skeleton structure for your top-level and documentation sections to modify as much as you like. The example site uses Docsy as a Git submodule so it's easy to [keep up to date](#keeping-the-theme-up-to-date). 
+You can clone the Docsy site using the command line or the Github UI.
 
-Don't forget when using your local copy to download the project's submodules (Docsy itself uses submodules for Bootstrap and Font Awesome) or you won't pull down some of the code you need to generate a working site.
+### Using the command line
 
 To copy the example site:
 
-1.  Make a local working copy of the example site directly using `git clone` (we won't get the submodules yet as this can confuse Git when setting our new origin repo in the next step):
+1.  Make a local working copy of the example site directly using `git clone`:
 
     ```bash
-    $ git clone https://github.com/google/docsy-example.git
+    git clone https://github.com/google/docsy-example.git
     ```
+1. In Github, create a [new empty repo](https://help.github.com/en/articles/create-a-repo) for your site. 
 
-    {{% alert title="Tip" %}}
-If you've already cloned the example site with `--recurse-submodules --depth 1` and want to set a new origin repo, worry not! The following is one way to un-confuse Git and you can then proceed with the next step as normal. You can remove the `old` remote once you've pushed to your repo.
-
-    git remote add old https://github.com/google/docsy-example.git
-    git fetch unshallow old
-
-    {{% /alert %}}
-
-1.  Then to store your site files in GitHub, create a new empty repo in your account and change your local copy's `origin` to your new repo (otherwise you'll be trying to push changes to our example site):
+1. Change your local copy's `origin` to your new repo (otherwise you'll be trying to push changes to the Docsy example site rather than to your own site):
 
     ```bash
     $ git remote remove origin
     $ git remote add origin https://github.com/my/example.git
     $ git push -u origin master
     ``` 
-  
+
 1. Get local copies of the project submodules so you can build and run your site locally:
 
     ```bash
     git submodule update --init --recursive
     ```
+1.  Switch to your `docsy-example` directory, and build your site:
+    
+    ```
+    hugo server
+    ```
+1.  Preview your site: http://localhost:1313/.
+    
 
-Alternatively, the following simple approach also works (though note this creates a link in GitHub between your project and the example site project):
+    {{% alert title="Tip" %}}
+If you've already cloned the example site with `--recurse-submodules --depth 1` and want to set a new origin repo, worry not! The following is one way to un-confuse Git and you can then proceed with the next step as normal. You can remove the `old` remote once you've pushed to your repo.
```

Comment:
(given that we've moved this)

you can then proceed with the next step as normal -> proceed to set the new origin

Reply:
done